deploy/nfpm/hoprd-sample.cfg.yaml

```diff
@@ -1,20 +1,16 @@
 ---
+db:
```

Comment:
the problem is that this did not prepackage the blokli prod URL, so none of the packages are runnable.